tools/depends/target/mariadb/Makefile

```diff
@@ -23,6 +23,7 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
 	cd $(PLATFORM); patch -p1 -i ../01-android.patch
 	cd $(PLATFORM); patch -p1 -i ../02-symbol_rename-gnutls_clash.patch
+	cd $(PLATFORM); patch -p1 -i ../03-cmake3.20_syntax_fix.patch
```

Comment:
did you forget to add that patch?

Reply:
Yeah, just woke up and out all morning, I'll get it fixed up this afternoon